gluten-core/src/main/scala/io/glutenproject/execution/WholeStageTransformer.scala

```diff
@@ -54,7 +54,11 @@ trait TransformSupport extends SparkPlan with LogLevelUtil {
 
   lazy val validateFailureLogLevel = GlutenConfig.getConf.validateFailureLogLevel
   lazy val printStackOnValidateFailure = GlutenConfig.getConf.printStackOnValidateFailure
+  var validateLog: Vector[String] = Vector()
 
+  def appendValidateLog (log: String): Unit = {
```

Comment:
I think this new method should be added in `GlutenPlan` to cover all gluten plans. `TransformSupport` only covers plans that can be put into wholestage transformer.

Reply:
Agree as I found there are nodes which are not inherited from `TransfromSupport`. It's good to add the interface in `GlutenPlan` as you said. Let me update.